pageObject_model/CalculatorHomePage.js

```diff
@@ -0,0 +1,182 @@
+        this.operatingSystemItemXpath =
+            '//md-option/*[contains(text(), "Free: Debian, CentOS, CoreOS, Ubuntu or BYOL (Bring Your Own License)")]';
+        this.machineClassSelectXpath =
+            '//*[@ng-model="listingCtrl.computeServer.class"]/*[@class="md-select-value"]//div';
+        this.machineClassListXpath = '//*[@id="select_container_93"]';
+        this.machineClassItemXpath = '//*[@id="select_option_90"]/div';
+        this.seriesSelectXpath =
+            '//*[@ng-model="listingCtrl.computeServer.series"]/*[@class="md-select-value"]//div';
+        this.seriesListXpath = '//*[@id="select_container_101"]';
+        this.seriesItemXpath = '//md-option/*[contains(text(), "N1")]';
+        this.machineTypeSelectXpath =
+            '//*[@ng-model="listingCtrl.computeServer.instance"]/*[@class="md-select-value"]//div';
+        this.machineTypeListXpath = '//*[@id="select_container_103"]';
+        this.machineTypeItemXpath =
+            '//md-option/*[contains(text(), "n1-standard-8 (vCPUs: 8, RAM: 30GB)")]';
+        this.checkboxXpath =
+            '//*[@aria-label="Add GPUs"]/div[@class="md-container md-ink-ripple"]';
+        this.numberGrusSelectXpath =
+            '//*[@placeholder="Number of GPUs"]/*[@class="md-select-value"]//div';
+        this.numberGrusListXpath = '//*[@id="select_container_454"]';
+        this.numberGrusItemXpath =
+            '//*[@id="select_option_462"]/div[@class="md-text ng-binding"]';
+        this.GruTypeSelectXpath = '//*[@aria-label="GPU type"]';
+        this.GruTypeListXpath = '//*[@value="NVIDIA_TESLA_K80"]/ancestor::div';
+        this.GruTypeItemXpath =
+            '//md-option/*[contains(text(), "NVIDIA Tesla V100")]';
+        this.localSsdSelectXpath =
+            '//*[@placeholder="Local SSD"]/*[@class="md-select-value"]//div';
+        this.localSsdListXpath = '//*[@id="select_container_414"]';
+        this.localSsdItemXpath = '//md-option/*[contains(text(), "2x375 GB")]';
+        this.datacenterSelectXpath =
+            '//*[@placeholder="Datacenter location"]/*[@class="md-select-value"]//div';
+        this.datacenterListXpath = '//*[@id="select_container_109"]';
+        this.datacenterItemXpath = '//*[@id="select_option_236"]/div[1]';
+        this.committedUsageSelectXpath =
+            '//*[@placeholder="Committed usage"]/*[@class="md-select-value"]//div';
+        this.committedUsageListXpath = '//*[@id="select_container_116"]';
+        this.committedUsageItemXpath = '//*[@id="select_option_113"]/div';
+        this.buttonAddToEstimateXpath =
+            '//button[@aria-label="Add to Estimate"]';
+    }
+
+    async selectSection() {
+        let checkedMode = await browser.$(this.checkedModeXpath);
+
+        await browser.waitUntil(elementToBeClickable(checkedMode), {
+            timeout: 10000,
+            timeoutMsg: 'Failed, after waiting for the element to be clickable',
+        });
+
```

Comment:
You can wrap these actions into some sepcisic function. Please see all the repo.